site/_docs/history.md

```diff
@@ -56,19 +146,297 @@ other software versions as specified in gradle.properties.
+  * [<a href="https://issues.apache.org/jira/browse/CALCITE-6116">CALCITE-6116</a>]
+   Add `EXISTS` function (enabled in Spark library)
+  * [<a href="https://issues.apache.org/jira/browse/CALCITE-6182">CALCITE-6182</a>]
+   Add `LENGTH`/`LEN` functions (enabled in Snowflake library)
+  * [<a href="https://issues.apache.org/jira/browse/CALCITE-6278">CALCITE-6278</a>]
+   Add `REGEXP`, `REGEXP_LIKE` function (enabled in Spark library)
+  * [<a href="https://issues.apache.org/jira/browse/CALCITE-6224">CALCITE-6224</a>]
+   Add `LOG2` function (enabled in MySQL, Spark library)
+  * [<a href="https://issues.apache.org/jira/browse/CALCITE-6223">CALCITE-6223</a>]
+   Add `MAP_CONTAINS_KEY` function (enabled in Spark library)
+  * [<a href="https://issues.apache.org/jira/browse/CALCITE-6205">CALCITE-6205</a>]
+   Add `BITAND_AGG`, `BITOR_AGG` functions (enabled in Snowflake library)
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6268">CALCITE-6268</a>]
+ Support implementing custom `JdbcSchema`
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6255">CALCITE-6255</a>]
+ Support BigQuery-style `JSON_OBJECT` invocation syntax
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6219">CALCITE-6219</a>]
+ 'Must-filter' columns
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-2980">CALCITE-2980</a>]
+ Implement the `FORMAT` clause of the `CAST` operator
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-3679">CALCITE-3679</a>]
+  Allow lambda expressions in SQL queries
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-3329">CALCITE-3329</a>]
+  Implement osquery for OS adapter
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-2040">CALCITE-2040</a>]
+  Create adapter for Apache Arrow
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6257">CALCITE-6257</a>]
+  StarRocks dialect implementation
 
 #### Dependency version upgrade
 {: #dependency-1-37-0}
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6124">CALCITE-6124</a>]
+ Bump json-path from 2.7.0 to 2.8.0
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6229">CALCITE-6229</a>]
+ Bump json-path from 2.8.0 to 2.9.0
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6378">CALCITE-6378</a>]
+ Bump Redis Docker image from 2.8.19 to 7.2.4
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6356">CALCITE-6356</a>]
+ Upgrade Calcite to Avatica 1.25.0
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6243">CALCITE-6243</a>]
+ Upgrade Cassandra to 4.1.3 and DataStax driver for Cassandra to 4.17.0
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6181">CALCITE-6181</a>]
+ Upgrade Janino from 3.1.9 to 3.1.11
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6119">CALCITE-6119</a>]
+ Upgrade testcontainers to 1.19.3
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6359">CALCITE-6359</a>]
+ Update GitHub Actions workflows to use docker compose v2
```

Comment:
This is is better to be put under "Build and test suite" section?

Reply:
yep i guess it could go there as well